File: app/src/main/java/com/neome/api/ent/base/dto/SpreadsheetFilterValueDateRange.java
// neome.ai API - do not change
//

package com.neome.api.ent.base.dto;

import org.jetbrains.annotations.Nullable;

import java.util.Date;

@SuppressWarnings("unused")
public class SpreadsheetFilterValueDateRange extends SpreadsheetFilterValue
{
  @Nullable
  public Date from;

  @Nullable
  public Date to;
}

File: app/src/main/java/com/neome/api/ent/base/dto/SpreadsheetFilterValueDateRange.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.ent.base.dto

import java.util.Date
import com.neome.api.ent.base.dto.SpreadsheetFilterValue

interface SpreadsheetFilterValueDateRange : SpreadsheetFilterValue
{
  val from: String?
  val to: String?
}